### DISCUSSION TITLE
运行npm run dev:weapp报错

### DISCUSSION THREAD

**Discussion by lowest-x** [NervJS/taro#15697] [Q & A]
拉别人的一个项目  下载了依赖包后运行 报错
![image](https://github.com/NervJS/taro/assets/105110808/bc137225-e148-4027-805f-2bb144d59608)

**Comment by ruochuan12** [comment]
如果是开源项目，能给到完整项目 `github` 地址更好。方便解答者复现问题~可能是 `node` 版本不一致导致的。
